modules/cluster-management/src/test/java/org/apache/ignite/internal/cluster/management/raft/AbstractRaftStorageTest.java

```diff
@@ -0,0 +1,248 @@
+
+        assertThat(storage.get(key), is(value1));
+
+        byte[] value2 = "value2".getBytes(UTF_8);
+
+        storage.put(key, value2);
+
+        assertThat(storage.get(key), is(equalTo(value2)));
+    }
+
+    /**
+     * Tests the {@link RaftStorage#remove} method.
+     */
+    @Test
+    void testRemove() {
+        byte[] key1 = "key1".getBytes(UTF_8);
+        byte[] key2 = "key2".getBytes(UTF_8);
+
+        byte[] value1 = "value1".getBytes(UTF_8);
+        byte[] value2 = "value2".getBytes(UTF_8);
+
+        storage.put(key1, value1);
+        storage.put(key2, value2);
+
+        storage.remove(key1);
+
+        assertThat(storage.get(key1), is(nullValue()));
+        assertThat(storage.get(key2), is(equalTo(value2)));
+    }
+
+    /**
+     * Tests that {@link RaftStorage#remove} works correctly if a value is missing.
+     */
+    @Test
+    void testRemoveNonExistent() {
+        byte[] key = "key".getBytes(UTF_8);
+
+        storage.remove(key);
+
+        assertThat(storage.get(key), is(nullValue()));
+    }
+
+    /**
+     * Tests the {@link RaftStorage#getWithPrefix} method.
+     */
+    @Test
+    void testGetWithPrefix() throws Exception {
+        storage.put("key1".getBytes(UTF_8), "value1".getBytes(UTF_8));
+        storage.put("key2".getBytes(UTF_8), "value2".getBytes(UTF_8));
+        storage.put("foo".getBytes(UTF_8), "value3".getBytes(UTF_8));
+
+        Cursor<String> cursor = storage.getWithPrefix("ke".getBytes(UTF_8), (k, v) -> new String(v, UTF_8));
+
+        try (cursor) {
+            assertThat(cursor.stream().collect(toList()), containsInAnyOrder("value1", "value2"));
+        }
+    }
+
+    /**
+     * Tests the {@link RaftStorage#getWithPrefix} method (corner case, when keys are close together lexicographically).
+     */
+    @Test
+    void testGetWithPrefixBorder() throws Exception {
+        byte[] key1 = "key1".getBytes(UTF_8);
+        byte[] key2 = key1.clone();
+
+        key2[key2.length - 1] += 1;
+
+        storage.put(key1, "value1".getBytes(UTF_8));
+        storage.put(key2, "value2".getBytes(UTF_8));
+
+        Cursor<String> cursor = storage.getWithPrefix(key1, (k, v) -> new String(v, UTF_8));
+
+        try (cursor) {
+            assertThat(cursor.stream().collect(toList()), containsInAnyOrder("value1"));
+        }
+    }
+
+    /**
+     * Tests that {@link RaftStorage#getWithPrefix} method works correctly over empty ranges.
+     */
+    @Test
+    void testGetWithPrefixEmpty() throws Exception {
+        storage.put("key1".getBytes(UTF_8), "value1".getBytes(UTF_8));
+        storage.put("key2".getBytes(UTF_8), "value2".getBytes(UTF_8));
+
+        Cursor<String> cursor = storage.getWithPrefix("foo".getBytes(UTF_8), (k, v) -> new String(v, UTF_8));
+
+        try (cursor) {
+            assertThat(cursor.stream().collect(toList()), is(empty()));
+        }
+    }
+
+    /**
+     * Tests the {@link RaftStorage#destroy()} method.
+     */
+    @Test
+    void testClear() {
```

Comment:
Let's rename the method to `testDestroy()` because it's `destroy()`, not `clear()`

Reply:
yes, that's an artifact, this method was called `clear` before